api/models/article.coffee

```diff
@@ -40,33 +35,55 @@ module.exports = class Article extends Backbone.Model
     else
       0
 
-  isEditorial: ->
+  isFeatured: ->
     @get('featured')
 
-  prepForInstant: ->
+  isEditorial: ->
+    @get('channel_id')?.toString() is EDITORIAL_CHANNEL
 
-    replaceTagWith = (htmlStr, findTag, replaceTag ) ->
-      $ = cheerio.load(htmlStr)
-      $(findTag).each ->
-        $(this).replaceWith($('<' + replaceTag + '>' + $(this).html() + '</' + replaceTag + '>'))
-      $.html()
+  prepForInstant: (cb) ->
+    @getAuthors (authors) =>
+      sections =  _.map @get('sections'), (section) =>
+        if section.type is 'text'
+          $ = cheerio.load(section.body)
+          $('br').remove()
+          $('*:empty').remove()
+          $('p').each ->
+            $(this).remove() if $(this).text().length is 0
+          section.body = $.html()
+          section.body = @replaceTagWith(section.body, 'h3', 'h2')
+          section
+        else if section.type in ['image_set', 'image_collection']
+          section.images = _.map section.images, (image) =>
+            if image.type is 'image'
+              image.caption = @replaceTagWith(image.caption, 'p', 'h1') if image.caption
+            image
+          section
+        else
+          section
+      @set
+        sections: sections
+        authors: authors
+      cb()
 
-    sections =  _.map @get('sections'), (section) ->
-      if section.type is 'text'
-        $ = cheerio.load(section.body)
-        $('br').remove()
-        $('*:empty').remove()
-        $('p').each ->
-          $(this).remove() if $(this).text().length is 0
-        section.body = $.html()
-        section.body = replaceTagWith(section.body, 'h3', 'h2')
-        section
-      else if section.type in ['image_set', 'image_collection']
-        section.images = _.map section.images, (image) ->
-          if image.type is 'image'
-            image.caption = replaceTagWith(image.caption, 'p', 'h1') if image.caption
-          image
-        section
+  replaceTagWith: (htmlStr, findTag, replaceTag) ->
+    $ = cheerio.load(htmlStr)
+    $(findTag).each ->
+      $(this).replaceWith($('<' + replaceTag + '>' + $(this).html() + '</' + replaceTag + '>'))
+    $.html()
+
+  getAuthors: (cb) ->
+    if @isEditorial()
+      return cb ['Artsy Editors'] unless @get('author_ids')?.length
+      Authors.mongoFetch
+        ids: @get('author_ids')
+      , (err, results) ->
```

Comment:
Would be cool to see these instances of callback-driven move to ES6 with async/await—but not something for this PR.